pkg/volume/util/goroutinemap/goroutinemap.go

```diff
@@ -0,0 +1,252 @@
+/*
```

Comment:
Done. Not sure what you mean by "move it out of goroutinemap directory"--this file is already in `pkg/volume/util`